packages/sanity/src/core/releases/tool/components/ReleasePublishAllButton/ReleasePublishAllButton.tsx

```diff
@@ -60,6 +62,12 @@ export const ReleasePublishAllButton = ({
       })
       // TODO: handle a published release on the document list
       router.navigate({})
+      if (
+        perspective.currentGlobalBundle !== 'published' &&
+        perspective.currentGlobalBundle?._id === release._id
+      ) {
+        perspective.setPerspective('drafts')
+      }
```

Comment:
Might be a mistake since I remember that we have made this change to remove it? This will make it that the pinned version will be removed once the publishing is done, is this something we want to do? 

It just feels like an oversight and unrelated to this PR :)

Reply:
I think we need this, I added it as a different PR but for some reason the change was removed? 
This was the PR I pushed https://github.com/sanity-io/sanity/pull/7925

If we keep the PR pinned when you visit a document the studio won't know how to properly resolve it.